src/javascript/app/pages/user/ico_claim_form.js

```diff
@@ -0,0 +1,60 @@
+const moment           = require('moment');
+const BinarySocket     = require('../../base/socket');
+const loadPreviousUrl  = require('../../base/binary_pjax').loadPreviousUrl;
+const Client           = require('../../base/client');
+const State            = require('../../../_common/storage').State;
+const getDecimalPlaces = require('../../common/currency').getDecimalPlaces;
+const toTitleCase      = require('../../../_common/string_util').toTitleCase;
+
+const ICOClaimForm = (() => {
+    const onLoad = () => {
+        const currency = Client.get('currency');
+        // Get ICO status.
+        const ico_req = {
+            ico_status: 1,
+            currency,
+        };
+        BinarySocket.send(ico_req);
+
+        BinarySocket.wait('get_settings', 'ico_status').then(() => {
+            const first_name     = toTitleCase(State.getResponse('get_settings.first_name'));
+            const last_name      = toTitleCase(State.getResponse('get_settings.last_name'));
+            const full_name      = `${first_name} ${last_name}`;
+            const decimal_places = getDecimalPlaces(currency);
+            const final_price    = (State.getResponse('ico_status.final_price')).toFixed(decimal_places);
```

Comment:
do we need to handle the case where `State.getResponse('ico_status.final_price')` is null?